tests/utils.py

```diff
@@ -80,22 +80,34 @@
 IS_NOT_ON_GITHUB = os.getenv("GITHUB_ACTIONS") != "true"
 # this env variable is set in regression test
 IS_IN_STORED_PROC_LOCALFS = IS_IN_STORED_PROC and os.getenv("IS_LOCAL_FS")
-# SNOW-1348805: Structured types have not been rolled out to all accounts yet.
-# Once rolled out this should be updated to include all accounts.
-STRUCTURED_TYPE_ENVIRONMENTS = {"dev", "aws"}
-IS_STRUCTURED_TYPES_SUPPORTED = (
-    os.getenv("cloud_provider", "dev") in STRUCTURED_TYPE_ENVIRONMENTS
-    and not IS_IN_STORED_PROC
-)
-ICEBERG_ENVIRONMENTS = {"dev", "aws"}
-IS_ICEBERG_SUPPORTED = os.getenv("cloud_provider", "dev") in ICEBERG_ENVIRONMENTS
 
 RUNNING_ON_GH = os.getenv("GITHUB_ACTIONS") == "true"
 RUNNING_ON_JENKINS = "JENKINS_HOME" in os.environ
 TEST_SCHEMA = f"GH_JOB_{(str(uuid.uuid4()).replace('-', '_'))}"
 if RUNNING_ON_JENKINS:
     TEST_SCHEMA = f"JENKINS_JOB_{(str(uuid.uuid4()).replace('-', '_'))}"
 
+# SNOW-1348805: Structured types have not been rolled out to all accounts yet.
+# Once rolled out this should be updated to include all accounts.
+STRUCTURED_TYPE_ENVIRONMENTS = {"SFCTEST0_AWS_US_WEST_2"}
+ICEBERG_ENVIRONMENTS = {"SFCTEST0_AWS_US_WEST_2"}
+
+
+def current_account(session):
+    return session.sql("select CURRENT_ACCOUNT_NAME()").collect()[0][0].upper()
```

Comment:
nit: we can use `session.get_current_account()` here?

Reply:
`session.get_current_account` would be the equivalent of `select CURRENT_ACCOUNT()` which is not exact enough for this use case. Eg for the `SFCTEST0_AWS_US_WEST_2` account it returns `SFCTEST0` instead of the full identifier.